tests/e2e/env-vars/00-install-instrumentation.yaml

```diff
@@ -4,4 +4,7 @@ metadata:
   name: sdk-only
 spec:
   exporter:
-    endpoint: http://localhost:4317
\ No newline at end of file
+    endpoint: http://localhost:4317
+  defaults:
+    useLabelsForResourceAttributes: true
```

Comment:
Please add a new test with this flag set, I'd like to preserve the test that uses the existing behavior

Reply:
done @TylerHelmuth